web-bluetooth/_includes/value-to-device-type.html

```diff
@@ -1,5 +1,5 @@
 <script>
-var valueToDeviceType = {
+const valueToDeviceType = {
```

Comment:
It also says `Never use the const keyword as it's not supported in Internet Explorer.`
In this case, the uppercase adds some emphasis on a simple utility object I'd like to keep as minimal as possible, hence the camelCase there.